packages/contracts-watr/contracts/common/XC20Wrapper.sol

```diff
@@ -0,0 +1,108 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.6.10;
+
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {Context} from "@openzeppelin/contracts/GSN/Context.sol";
+import {SafeMath} from "@openzeppelin/contracts/math/SafeMath.sol";
+import {Address} from "@openzeppelin/contracts/utils/Address.sol";
+
+import {ClaimableOwnable} from "./ClaimableOwnable.sol";
+import {IERC20Plus} from "../interface/IERC20Plus.sol";
+
+abstract contract XC20Wrapper is IERC20, ClaimableOwnable, Context {
+    using SafeMath for uint256;
+
+    function _mint(address account, uint256 amount) internal virtual {
+        require(account != address(0), "XC20: mint to the zero address");
+        IERC20Plus(nativeToken).mint(account, amount);
+        emit Transfer(address(0), account, amount);
+    }
+
+    function _burn(address account, uint256 amount) internal virtual {
+        IERC20Plus(nativeToken).burn(account, amount);
+        emit Transfer(account, address(0), amount);
+    }
+
+    function decimals() public view virtual returns (uint8) {
+        return IERC20Plus(nativeToken).decimals();
```

Comment:
It is later overwritten by `18` in `TrueUSD.sol` is that okay? Maybe we don't need to call this method on `nativeToken` or should not override in `TrueUSD.sol`?